functions/Copy-DbaDbViewData.ps1

```diff
@@ -219,10 +219,10 @@ function Copy-DbaDbViewData {
             }
         }'
 
-        Add-Type -ReferencedAssemblies System.Data.dll -TypeDefinition $sourcecode -ErrorAction Stop
+        Add-Type -ReferencedAssemblies System.Data.SqlClient.dll -TypeDefinition $sourcecode -ErrorAction Stop
```

Comment:
Which DLL location will this be picked up from?

Reply:
@potatoqualitee should we enforce this getting the DLL from our bin path based on version of PS?

Reply:
It took me a while to work out how the libraries are loaded.  Powershell 6 appears to require a different partialname for the library to be referenced.

Reply:
We always load with the explicit path of the DLL.  Providing that with no DLL means it will look in the GAC of the machine and not use the version we have in the module. So you can get different results if someone has a newer version of that DLL, not likely but it has the potential. (We have had other issues where the GAC version of SMO got loaded in some cases and it causes all kinds of issues because we do not have some features because of the version of SMO we are using.)

I can't recall but we have other commands that do Add-Type that might show how we explicitly load DLL.